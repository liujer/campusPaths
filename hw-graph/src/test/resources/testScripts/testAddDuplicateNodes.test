# Tests adding duplicate nodes to a graph
CreateGraph B
AddNode B one
AddNode B two
AddNode B two
ListNodes B