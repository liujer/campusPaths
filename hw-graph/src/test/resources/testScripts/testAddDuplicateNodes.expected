# Tests adding duplicate nodes to a graph
Created graph B
Added node one to B
Added node two to B
Could not add node two to B
ListNodes of B output: one two